scripts/scil_bundle_volume.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import argparse
+import json
+import os
+
+import nibabel as nib
+import numpy as np
+
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_json_args,
+                             add_reference_arg,
+                             assert_inputs_exist)
+from scilpy.tractanalysis import compute_tract_counts_map
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description='Compute bundle volume in mm³',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    p.add_argument('in_bundle',
+                   help='Fiber bundle file.')
+
+    add_reference_arg(p)
+    add_json_args(p)
+
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, [args.in_bundle, args.reference])
```

Comment:
args.reference is now optional, will crash most often than not (since --reference is only required for tck)